scripts/run-pep8.sh

```diff
@@ -10,7 +10,7 @@ pep8 --config=.pep8 openassessment > $PEP8_VIOLATIONS
 NUM_PEP8_VIOLATIONS=$(cat $PEP8_VIOLATIONS | wc -l)
 
 echo "Found" $NUM_PEP8_VIOLATIONS "pep8 violations, threshold is" $MAX_PEP8_VIOLATIONS
-if [[ $NUM_PEP8_VIOLATIONS > $MAX_PEP8_VIOLATIONS ]]; then
+if [[ $NUM_PEP8_VIOLATIONS -gt $MAX_PEP8_VIOLATIONS ]]; then
```

Comment:
Note to reviewers: apparently ">" does something different than greater than. It appeared to work with the number I tried, but then I hit a number that didn't work. 

I don't have experience writing bash scripts...

http://www.tldp.org/LDP/abs/html/comparison-ops.html